Sources/PrimerSDK/Classes/Core/Primer/AppState.swift

```diff
@@ -48,7 +50,17 @@ internal class AppState: AppStateProtocol {
     var authorizationToken: String?
     var customerToken: String?
     var sessionId: String?
+    private var apayaResult: Result<ApayaWebViewResult, ApayaException> = .failure(ApayaException.webViewFlowNotComplete)
```

Comment:
This could be optional and be initialized with `nil` value. I know that you init it with `webViewFlowNotComplete` value, but in the case of not having Apaya enabled this doesn't make a lot of sense. I think that `nil` is more suitable since it means that it wasn't initialized.

Reply:
Could definitely change this param, but wouldn't this mean we have to add a lot of `guard` statements rather than rely purely on switch case?